src/Bluejay.asm

```diff
@@ -761,11 +761,10 @@ motor_start:
     mov  Flags1, A
```

Comment:
Unrelated to this PR, but why do we `clr A` and then `mov` A into all other vars instead of `clr` them. Both instructions take one cycle and a `clr` would be more implicit, no?

Reply:
clr can only be used with A and bits

Changed everything to move 0 as literal to make it more explicit